heroku/cli
# Add nyc with working script & update .gitignore

## Why the change?
[GUS Card](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001NBcxsYAD/view)

As per our review of unit tests and coverage for the apps-v5 package, this PR does the following:
- Adds `nyc` package to add code coverage reports ✅
- Ensures that our code coverage reports over 85% test coverage of statements ✅

## Additional Notes
- No additional test failure scenarios to include. Incorrect user inputs have been accounted for.

## How to verify?
1. Pull down branch
2. Run tests
3. Confirm test coverage report is visible and that it reports over 85% test coverage of statements
